articles/aks/monitor-apiserver.md

```diff
@@ -22,7 +22,7 @@ Kubernetes audit logging isn't enabled by default on an AKS cluster on account o
 * **Destination details:** Select the checkbox for **Log Analytics**.
 
 > [!NOTE]
-> There could be substantial cost involved once kube-audit logs are enabled. Consider disabling kube-audit logging when not required.
+> There could be substantial cost involved once kube-audit logs are enabled. Consider disabling kube-audit logging when not required. If you're okay with only PUT and DELETE-style requests getting logged (meaning minimal GET requests), then kube-audit-admin instead of kube-audit is one way to help reduce cost.
```

Comment:
Suggest rewording to - An alternative approach to significantly reduce the number of logs and help reduce cost is by enabling collection from kube-audit-admin, which excludes the get and list audit events.

Reply:
Thanks, updated